docs/java-rest/high-level/ml/put-data-frame-analytics.asciidoc

```diff
@@ -96,6 +96,10 @@ include-tagged::{doc-tests-file}[{api}-outlier-detection-customized]
 <1> Constructing a new OutlierDetection object
 <2> The method used to perform the analysis
 <3> Number of neighbors taken into account during analysis
+<4> The min `outlier_score` required to compute feature influence
```

Comment:
Just curious: Could the functionality of `compute_feature_influence` setting be achieved with setting `min_outlier_score` to a *very high* number?

Reply:
Indeed, one can set `feature_influence_threshold` to `1` to achieve the same as setting `compute_feature_influence` to `false`.